testsuite/MDAnalysisTests/coordinates/test_chainreader.py

```diff
@@ -206,6 +208,17 @@ def test_set_all_format_tuples(self):
         assert universe.trajectory.n_frames == 21
         assert_equal(universe.trajectory.filenames, [PDB, XTC, TRR])
 
+    def test_set_all_format_lammps(self):
+        universe = mda.Universe(
+            LAMMPS_chain, [LAMMPSDUMP_chain1, LAMMPSDUMP_chain2], 
+            format="LAMMPSDUMP", continuous=True)
+        assert universe.trajectory.n_frames == 11
+
+        # Test whether the amount of time between frames is consistently = 1
+        time_values = np.array([ts.time for ts in universe.trajectory])
+        dt_array = time_values[1:]-time_values[:-1]
```

Comment:
Could you also check that the times are `[0,1,2,3,4,5,6,7]` using a qucik `np.arange(11)`? I think I am getting that right?